gitbook-docs/market/user-guides/circuit-developer.md

```diff
@@ -1,51 +1,64 @@
-# Circuit Developer
+# Circuit developer
 
-This is an entity that prepares circuits for the proof market. The most efficient way to create a circuit definition for Proof Market is [zkLLVM](https://github.com/NilFoundation/zkllvm). All dependencies and build instructions are inside the [zkLLVM's repository](https://github.com/NilFoundation/zkllvm).
+This is an entity that prepares circuits for Proof Market.
+The most efficient way to create a circuit definition for Proof Market is
+[zkLLVM](https://github.com/NilFoundation/zkllvm) circuit compiler.
+You can find the required dependencies and build instructions in the
+[zkLLVM's repository](https://github.com/NilFoundation/zkllvm).
 
-Anyone can generate circuits. They are serialised & published on the proof market. This allows the reuse of the circuits by all other proof requesters.&#x20;
+Anyone can generate circuits.
+They are serialized & published on Proof Market, allowing reuse of the circuits by
+all proof requesters.
 
-## Create a new circuit (using [zkLLVM](https://github.com/NilFoundation/zkllvm))
+## Create a new circuit using [zkLLVM](https://github.com/NilFoundation/zkllvm)
 
-```
+```bash
 make -C ${ZKLLVM_BUILD:-build} <circuit target name> -j$(nproc)
 ```
 
 ## Prepare a statement with a circuit description for Proof Market
 
-Circuits are stored as a _statement_ structure on Proof Market. Statement description example can be found in /example directory
+Circuits are stored as a _statement_ structure on Proof Market.
+Example of a statement description can be found in the `/example` directory.
```

Comment:
We're still struggling with terminology.

What's a statement structure? What's a statement description?

I understand that a statement is a data structure internally, but it should not bother circuit developers. For them, it's just an artifact that is produced from a circuit and accepted by Proof Market.

By its meaning and purpose, a statement is a description of a circuit combined with the circuit itself. But in the `./example` directory we see the statements, not their descriptions.

So, let's reduce the number of used terms. Just statements. 

```suggestion
Circuits are stored on Proof Market in the form of statements.
Example statements can be found in the `./example` directory.
```

Oh, and paths like `/path` are paths from the root of filesystem or a website. Paths from the project root or current directory are `./path/`, `./path` or `path/`. Just `path.txt` can be okay for files, but not so much for directories.

Reply:
Fixed

Reply:
About paths - we should probably check and fix other PMT's docs